practice-one/src/styles/variables.css

```diff
@@ -1,11 +1,14 @@
 :root {
   --color-dark: #191919;
   --color-primary: #F54748;
+  --color-icon-wrapper:#FEE5E5;
   --color-accent: #FDC55E;
   --color-light: #FFFFFF;
   --color-neutral: #C4C4C4;
   --color-warning: #FFB800;
   --color-gradient-warm: linear-gradient(0deg, rgba(245, 71, 72, 0) 57.72%, #FDC55E 112.46%);
   --color-gradient-strong: linear-gradient(179.69deg, rgba(255, 255, 255, 0) , #F54748 303.75%);
   --font-family-primary: 'Sofia Pro Regular';
+  --font-family-dark: 'Sofia Pro Black';
+  --font-family-search-box: 'Poppins Medium';
```

Comment:
This is wrong, you no need to define another fontfamily everytime fontweight change

Reply:
all you need is
```
--font-family-primary: 'Sofia'
--font-family-secondary: 'Poppins'

.example-class {
  font-family: var(--font-family-primary);
  font-weight: medium
}
```

Reply:
i updated it